h/search/index.py

```diff
@@ -26,7 +27,17 @@ class Window(namedtuple('Window', ['start', 'end'])):
     pass
 
 
-def index(es, annotation, request, target_index=None):
+def index(models_annotation, request, index="hypothesis"):
```

Comment:
No `es` (client) argument needed here. The only reason `request` is needed is for sending a notification.